ll-token-field.html

```diff
@@ -110,18 +167,32 @@
         e.stopPropagation();
         var newTag = input.value;
         input.value = '';
-        this.tags = this.tags.concat(newTag);
+        this.push('tags', newTag);
+        this.tagChanged();
         input.focus();
       }
       else if (e.keyCode === 8 && input.value === '') { // Backspace
```

Comment:
I know this condition was already here, and it's my fault, but we should probably make sure the string isn't whitespace as well as empty so we don't end up with empty tags.